prettier formatting not working with lint staged

## Summary

Added Prettier formating in #24 which should be running Prettier on commit.  However, as just recently observed in #50 , the [build was failing](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/actions/runs/9238084134/job/25415767141?pr=50) on the `npm run format:check` task on the root level _CONTRIBUTING.md_
```sh
Run npm run format:check

> www.greenwoodjs.dev@0.0.1 format:check
> prettier . --check

Checking formatting...
[warn] CONTRIBUTING.md
[warn] Code style issues found in the above file. Run Prettier to fix.
Error: Process completed with exit code 1.
```

but that should already be running as this is what we have in our [_package.json_](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/blob/main/package.json#L73) right?
```json
"lint-staged": {
  "*.js": "npm run lint:js",
  "*.css": "npm run lint:css",
  "**/*": "npm run lint:ls && npm run format"
}
```

## Details

Can confirm that after running `npm run format` locally, the file showed up as having needed formatting.  🤷‍♂️ 
```sh
➜  www.greenwoodjs.dev git:(docs/issue-7-styling-contributing-docs) ✗ npm run format

> www.greenwoodjs.dev@0.0.1 format
> prettier . --write

.github/ISSUE_TEMPLATE.md 11ms (unchanged)
.github/PULL_REQUEST_TEMPLATE.md 2ms (unchanged)
.github/workflows/ci.yml 13ms (unchanged)
.ls-lint.yml 2ms (unchanged)
.prettierrc 17ms (unchanged)
.storybook/main.js 5ms (unchanged)
.storybook/preview.js 3ms (unchanged)
CONTRIBUTING.md 50ms. # <-- it got formatted!!
eslint.config.js 16ms (unchanged)
FEEDBACK.md 12ms (unchanged)
greenwood.config.js 1ms (unchanged)
package-lock.json 92ms (unchanged)
package.json 23ms (unchanged)
plugin-css-modules-bak.js 14ms (unchanged)
plugin-css-modules-log-bak.js 32ms (unchanged)
plugin-css-modules.js 11ms (unchanged)
README.md 4ms (unchanged)
```

--- a/lint-staged.config.js
+++ b/lint-staged.config.js
@@ -1,5 +1,5 @@
 export default {
-  "*.js": ["npm run lint:js --", "npm run format --"],
+  "*.js": ["npm run lint:js --"],
   "*.css": "npm run lint:css --",
   "*.*": ["npm run lint:ls --", "npm run format --"],
 };

--- a/lint-staged.config.js
+++ b/lint-staged.config.js
@@ -1,0 +1,5 @@
+export default {
+  "*.js": ["npm run lint:js --", "npm run format --"],
+  "*.css": "npm run lint:css --",
+  "*.*": ["npm run lint:ls --", "npm run format --"],
+};

--- a/package.json
+++ b/package.json
@@ -81,8 +81,11 @@
     "@rollup/rollup-linux-x64-gnu": "^4.17.2"
   },
   "lint-staged": {
-    "*.js": "npm run lint:js",
-    "*.css": "npm run lint:css",
-    "**/*": "npm run lint:ls && npm run format"
+    "*.js": [
+      "eslint -c ./eslint.config.js",
+      "prettier . --write"
+    ],
+    "*.css": "stylelint \"./src/**/*.css\"",
+    "*.*": "ls-lint"
   }
 }

--- a/package.json
+++ b/package.json
@@ -74,18 +74,10 @@
     "storybook": "^8.0.6",
     "storybook-addon-fetch-mock": "^2.0.1",
     "stylelint": "^16.4.0",
-    "stylelint-prettier": "^5.0.0",
+    "stylelint-config-recommended": "^14.0.1",
     "vite": "^5.2.8"
   },
   "optionalDependencies": {
     "@rollup/rollup-linux-x64-gnu": "^4.17.2"
-  },
-  "lint-staged": {
-    "*.js": [
-      "npm run lint:js --",
-      "npm run format --"
-    ],
-    "*.css": "npm run lint:css --",
-    "*.*": "npm run lint:ls --"
   }
 }

--- a/package.json
+++ b/package.json
@@ -82,10 +82,10 @@
   },
   "lint-staged": {
     "*.js": [
-      "eslint -c ./eslint.config.js",
-      "prettier . --write"
+      "npm run lint:js --",
+      "npm run format --"
     ],
-    "*.css": "stylelint \"./src/**/*.css\"",
-    "*.*": "ls-lint"
+    "*.css": "npm run lint:css --",
+    "*.*": "npm run lint:ls --"
   }
 }

--- a/src/components/capabilities/capabilities.css
+++ b/src/components/capabilities/capabilities.css
@@ -9,22 +9,9 @@
   font-size: var(--font-size-1);
 }
 
-/* consolidated below */
-/* .heading {
-  font-family: var(--font-primary-bold);
-  font-size: var(--font-size-6);
-} */
-
 .container {
   padding: var(--size-fluid-2);
 }
-
-/* consolidated below */
-/* .sections {
-  overflow-x: auto;
-  scrollbar-width: thin;
-  scrollbar-color: var(--color-gray) transparent;
-} */
 
 .section {
   display: inline-block;

--- a/src/components/capabilities/capabilities.css
+++ b/src/components/capabilities/capabilities.css
@@ -9,20 +9,22 @@
   font-size: var(--font-size-1);
 }
 
-.heading {
+/* consolidated below */
+/* .heading {
   font-family: var(--font-primary-bold);
   font-size: var(--font-size-6);
-}
+} */
 
 .container {
   padding: var(--size-fluid-2);
 }
 
-.sections {
+/* consolidated below */
+/* .sections {
   overflow-x: auto;
   scrollbar-width: thin;
   scrollbar-color: var(--color-gray) transparent;
-}
+} */
 
 .section {
   display: inline-block;
@@ -91,6 +93,8 @@
   border-top: 1px solid var(--color-accent);
   border-bottom: 1px solid var(--color-accent);
   margin: var(--font-size-1) 0;
+  scrollbar-width: thin;
+  scrollbar-color: var(--color-gray) transparent;
 }
 
 .sections-list {

--- a/src/components/capabilities/capabilities.css
+++ b/src/components/capabilities/capabilities.css
@@ -42,8 +42,8 @@
   border: 2px dotted var(--color-accent);
 }
 
-.section:hover strong,
-.active strong {
+.active strong,
+.section:hover strong {
   font-family: var(--font-primary-bold);
   color: var(--color-white);
 }

--- a/src/components/copy-to-clipboard/copy-to-clipboard.css
+++ b/src/components/copy-to-clipboard/copy-to-clipboard.css
@@ -1,5 +1,4 @@
 #icon {
-  background-color: transparent;
   padding: var(--size-2);
   border: none;
   cursor: pointer;

--- a/src/components/footer/footer.module.css
+++ b/src/components/footer/footer.module.css
@@ -10,7 +10,6 @@
   padding-left: var(--size-fluid-1);
   display: flex;
   flex-direction: column;
-  flex-basis: 100%;
   flex: 1;
   justify-content: center;
   color: var(--color-gray);

--- a/src/components/header/header.module.css
+++ b/src/components/header/header.module.css
@@ -45,6 +45,11 @@
   color: var(--color-black);
 }
 
+.mobileMenuListItem a {
+  color: var(--color-black);
+  text-decoration: none;
+}
+
 .navBarMenuItem a:hover {
   text-decoration: underline;
 }
@@ -108,10 +113,10 @@
   font-size: var(--font-size-5);
 }
 
-.mobileMenuListItem a {
+/* .mobileMenuListItem a {
   color: var(--color-black);
   text-decoration: none;
-}
+} */
 
 @media screen and (min-width: 480px) {
   .container {

--- a/src/components/header/header.module.css
+++ b/src/components/header/header.module.css
@@ -113,11 +113,6 @@
   font-size: var(--font-size-5);
 }
 
-/* .mobileMenuListItem a {
-  color: var(--color-black);
-  text-decoration: none;
-} */
-
 @media screen and (min-width: 480px) {
   .container {
     justify-content: space-between;

--- a/src/components/hero-banner/hero-banner.module.css
+++ b/src/components/hero-banner/hero-banner.module.css
@@ -84,11 +84,9 @@
 
 .snippet pre {
   font-size: 16px;
-  display: inline-block;
   height: 46px;
   align-content: center;
   display: inline;
-  vertical-align: text-top;
   background-color: transparent;
   color: var(--color-black);
   vertical-align: middle;

--- a/src/components/side-nav/side-nav.module.css
+++ b/src/components/side-nav/side-nav.module.css
@@ -8,10 +8,6 @@
     color: var(--color-black);
     text-decoration: none;
   }
-
-  & a:hover {
-    opacity: 0.7;
-  }
 }
 
 .compactMenuCloseButton {
@@ -54,6 +50,19 @@
   height: auto;
 }
 
+.compactMenuSectionListItemActive {
+  & a {
+    border-left: var(--size-1) solid black;
+    border-radius: var(--radius-2);
+    background-color: white;
+    padding: 0 var(--size-2);
+    min-width: 200px;
+    display: inline-block;
+    box-shadow: var(--shadow-3);
+    margin-left: -16px;
+  }
+}
+
 .compactMenuSectionHeading {
   margin: 0 0 var(--size-2) 0;
 
@@ -64,6 +73,7 @@
   & a.active,
   & a:hover {
     text-decoration: underline;
+    opacity: 0.7;
   }
 
   & a.active::before {
@@ -77,19 +87,6 @@
 
   & a:hover {
     opacity: 0.7;
-  }
-}
-
-.compactMenuSectionListItemActive {
-  & a {
-    border-left: var(--size-1) solid black;
-    border-radius: var(--radius-2);
-    background-color: white;
-    padding: 0 var(--size-2);
-    min-width: 200px;
-    display: inline-block;
-    box-shadow: var(--shadow-3);
-    margin-left: -16px;
   }
 }
 

--- a/src/components/side-nav/side-nav.module.css
+++ b/src/components/side-nav/side-nav.module.css
@@ -60,9 +60,7 @@
   & a {
     font-family: var(--font-primary-bold);
   }
-}
 
-.compactMenuSectionHeading {
   & a.active,
   & a:hover {
     text-decoration: underline;

--- a/stylelint.config.js
+++ b/stylelint.config.js
@@ -1,6 +1,3 @@
 export default {
-  // no stylelint 16 support yet
-  // https://github.com/developer-stylechain/stylelint-a11y/issues/4
-  // plugins: ['stylelint-a11y'],
-  extends: ["stylelint-prettier/recommended"],
+  extends: ["stylelint-config-recommended"],
 };

